proto/proto-backplane-grpc/src/main/proto/deephaven/proto/console.proto

```diff
@@ -46,11 +46,11 @@ message GetConsoleTypesResponse {
 }
 
 message StartConsoleRequest {
-    arrow.flight.protocol.Ticket result_id = 1;
+    io.deephaven.proto.backplane.grpc.Ticket result_id = 1;
```

Comment:
should we drop the prefixes instead? they should be in the same namespace now

Reply:
Nope, console is in .script.grpc, since it is a different package it needs qualified imports (same as ETCR references in this file)

Reply:
I wonder if it make sense to be in another package anymore. Especially if it gets the package convention from the file structure in python. Either way, we can table this for now. Just food for thought.